test/unit/vl-tabs.test.html

```diff
@@ -126,9 +138,15 @@
             setTimeout(() => {
               assert.lengthOf(getTabArray(), 3);
               assert.lengthOf(getTabSectionArray(), 3);
-              assert.deepEqual(getTabArray().map((tab) => tab.id), ['1', '2', '3']);
-              assert.deepEqual(getTabSectionArray().map((tabSection) => tabSection.querySelector('slot').assignedNodes()[0].innerText), ['1 content', '2 content', '3 content']);
-              done();
+              element.insertAdjacentHTML('beforeend', `<vl-tabs-pane data-vl-id="4"><span slot="4-title-slot">4</span>4 content</vl-tabs-pane>`);
+              setTimeout(() => {
+                assert.lengthOf(getTabArray(), 4);
+                assert.lengthOf(getTabSectionArray(), 4);
+                assert.deepEqual(getTabArray().map((tab) => tab.id), ['1', '2', '3', '4']);
+                assert.deepEqual(getTabSectionArray().map((tabSection) => tabSection.querySelector('slot').assignedNodes()[0].innerText), ['1 content',
+                  '2 content', '3 content', '4 content']);
```

Comment:
@coemans 
als we in src/vl-tabs.js `tabPaneTitleSlot.remove();` verwijderen. Zal deze test falen de innerText van de 4de tabsectie zal dan '44 content' verwachten.
Dit klopt de text van het title slot meegenomen wordt. Willen we dit side-effect? Anders mag voor mij die remove er ook uit.

Reply:
test case aangepast nadat het slot niet meer verwijdert wordt